pkg/routeagent/controllers/route/route.go

```diff
@@ -310,6 +326,43 @@ func (r *Controller) updateIptableRulesForInterclusterTraffic(inputCidrBlocks []
 	}
 }
 
+func (r *Controller) updateRoutingRulesForHostNetworkSupport(inputCidrBlocks []string, operation Operation) {
+	if operation == FlushRouteTable {
+		r.routeCacheGWNode.DeleteAll()
+		cmd := exec.Command("/sbin/ip", "r", "flush", "table", strconv.Itoa(RouteAgentHostNetworkTableID))
+		if err := cmd.Run(); err != nil {
+			// We can safely ignore this error, as this table
+			// won't exist in most nodes (only gateway nodes)
```

Comment:
I think we should at least log it to debug. Also can we distinguish between the table non-existent and another failure?

Reply:
Actually, we cannot do anything and we will not have any benefit if we log the error message. So, it's okay to ignore it.

Reply:
I assume it  could fail for another reason. In my experience it's better to log an error rather than silence it.  Ideally if we can distinguish table non-existent error but at least log it to debug.

Reply:
Ok